config/config-reporting/models.go

```diff
@@ -0,0 +1,5 @@
+package config_reporting
```

Comment:
avoid `_` in your package names.

```suggestion
package reporting
```